src/App.tsx

```diff
@@ -15,37 +16,85 @@
   'Garlic',
 ];
 
+enum Way {
+  'Unknown' = 0,
+  'Info' = 'is-info',
+  'Success' = 'is-success',
+}
+
+function getSorted(obj: string[], way: Way) {
+  const currentGoods = [...obj];
+
+  currentGoods.sort((el1, el2) => {
+    switch (way) {
+      case 'is-info':
+        return el1.localeCompare(el2);
+      case 'is-success':
+        return el1.length - el2.length;
+      default:
+        return 0;
+    }
+  });
+
+  return currentGoods;
+}
+
 export const App: React.FC = () => {
+  const [selectItem, setSelectItem] = useState('');
+  const [selReverse, setselReverse] = useState('');
```

Comment:
The `selReverse` state is used to determine if the goods should be reversed, but it is initialized as a string. It would be more appropriate to use a boolean for this state, e.g., `useState(false)`.